### ISSUE TITLE
Have a look at FastCDC

### ISSUE THREAD

**Issue by titusz** [netleibi/fastchunking#2]
I have been playing around with `fastchunking` and it works really well. As the name suggests this lib is aimed at **fast** chunking. So I´d like to bring the [FastCDC paper](https://www.usenix.org/node/196197) to attention. It claims to be about 10x faster than Rabin-based CDCs. While I did not verify this claim in practice I read the paper and the approach seems promising. I´d be interested in thoughts about this.

**Comment by netleibi** [comment]
I agree that their approach is promising and that it would be interesting to have it included in this library. I am not aware of any existing FastCDC implementation, though, as the authors have not published their source code yet, so I think it's something to have a look at in the future.
